haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -2,7 +2,75 @@
 -- SPDX-License-Identifier: MIT
 
 module Lorentz.Contracts.Stablecoin
-    (
-    ) where
+  ( Parameter
+  , Storage
+  , mkPermissionDescriptor
+  , mkTokenMetadata
+  ) where
+
+import Text.Show (Show)
+import Lorentz
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
```

Comment:
Let's put it in this repo for now. It may be not so neat (because later we will move it back), but it should speed things up.

Reply:
Moved here.